apps/labs/posts/joining_labs.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Thoughts on joining Quansight Labs'
+published: April 8, 2019
+author: ralf-gommers
+description: ''
```

Comment:
```suggestion
description: 'Ralf Gommers's perspectives for pushing forward the Python ecosystem for scientific computing and data science, and how to fund it.'
```

Reply:
@rgommers Is this a good description here? If not, maybe you can suggest an alternative?

Reply:
How about: Ralf Gommers's thoughts on joining Quansight Labs as its director, and his perspectives for pushing forward the Python ecosystem for scientific computing and data science, and how to fund it.

Reply:
Thanks!